pkg/install/condition.go

```diff
@@ -62,6 +63,11 @@ func (i *Installer) clusterVersionReady() (bool, error) {
 func (i *Installer) ingressControllerReady() (bool, error) {
 	ic, err := i.operatorcli.OperatorV1().IngressControllers("openshift-ingress-operator").Get("default", metav1.GetOptions{})
 	if err == nil && ic.Status.ObservedGeneration == ic.Generation {
+		if _, ok := i.env.(env.Dev); !ok {
+			if ic.Spec.DefaultCertificate == nil {
+				return false, nil
+			}
+		}
```

Comment:
You could consolidate this:
```
		if _, ok := i.env.(env.Dev); !ok && ic.Spec.DefaultCertificate == nil {
			return false, nil
		}
```

Reply:
Thanks.  I felt like the previous version was more readable.